llvm/test/CodeGen/X86/tls-function-argument.ll

```diff
@@ -0,0 +1,17 @@
+; RUN: llc -verify-machineinstrs < %s -relocation-model=pic
+
+target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-unknown-linux-gnu"
```

Comment:
This can put in the RUN line like
`; RUN: llc -mtriple=x86_64 -verify-machineinstrs < %s -relocation-model=pic | FileCheck %s`

And add FileCheck to show the assembly.

Reply:
Yes, we need to check the functional correctness.

Reply:
Done in the updated commit.